app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewsController < ApplicationController
+  before_action :authorize
+
+  def create
+    review = Review.new review_params
+
+    if review.save
+      redirect_to book_path book,
```

Comment:
I think the lack of parens is messing this up, the flash message isn't appearing – the text ends up in the query string.